pontoon/actionlog/migrations/0004_auto_20240816_1045.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 4.2.11 on 2024-08-16 10:45
+
+import datetime
+
+from django.db import migrations
+
+
+def migrate_translation_to_actionlog(apps, schema_editor):
+    Translation = apps.get_model("base", "Translation")
+    ActionLog = apps.get_model("actionlog", "ActionLog")
+
+    traslation_info = {
+        "date": ActionLog.ActionType.TRANSLATION_CREATED,
+        "approved_date": ActionLog.ActionType.TRANSLATION_APPROVED,
+        "unapproved_date": ActionLog.ActionType.TRANSLATION_UNAPPROVED,
+        "rejected_date": ActionLog.ActionType.TRANSLATION_REJECTED,
+        "unrejected_date": ActionLog.ActionType.TRANSLATION_UNREJECTED,
+    }
+
+    # date to end migration
+    end_date = datetime.datetime(
+        2020, 1, 7, 9, 25, 11, 829125, tzinfo=datetime.timezone.utc
+    )
+
+    actions_to_log = []
+
+    for translation in Translation.objects.filter(date__lt=end_date):
+        for attr, action_type in traslation_info.items():
+            if getattr(translation, attr) is not None:
+                actions_to_log.append(
+                    ActionLog(
+                        action_type=action_type,
+                        created_at=translation.date,
+                        entity_id=translation.entity_id,
+                        locale_id=translation.locale_id,
```

Comment:
We should remove lines for these two fields. They are only used [when a translation has been deleted or a team comment has been added](https://github.com/mozilla/pontoon/blob/4fe3b6bd8d1cbbeae34e5cbbb8d582a2cedef35f/pontoon/actionlog/models.py#L36), neither of which is relevant for this migration.